src/ripple/app/main/Application.cpp

```diff
@@ -847,11 +847,11 @@ class ApplicationImp
         //      things will happen.
         m_resolver->stop ();
 
-        if(startTimers_)
-        {
-            m_sweepTimer.cancel ();
-            m_entropyTimer.cancel ();
-        }
+        sweepTimer_.cancel ();
```

Comment:
It may be worth using the `cancel` overload with an error_code and log on failure.
eg.
```        boost::system::error_code ec;
        sweepTimer_.cancel(ec);
        if (ec)
        {
            JLOG (j_.trace()) << "Application: sweepTimer cancel error: " << ecCancel.message ();
        }```